utils/manifests/definitions.go

```diff
@@ -30,5 +30,10 @@ type CrdFilter struct {
 	GroupFilter   JsonPath //This will specify the path to get to group name
 	VersionFilter JsonPath //This will specify the path to get to version name. [Version should have a name field]
 	SpecFilter    JsonPath //This will specify the path to get spec
-	IsJson        bool     //Set to true if input format is json
+	ItrFilter     string   //Filter is appended with "=="<some-crd-name>")]", so no need to write it completely. Just write the part before double equals
```

Comment:
What does Itr stand for?

Reply:
Iterator
It Iterates over the objects and picks only the one we are interested in